mix.exs

```diff
@@ -44,18 +44,23 @@ defmodule IncidentIo.Mixfile do
       description: "An Elixir client for the incident.io API.",
       maintainers: ["Sasha Gerrand"],
       licenses: ["MIT"],
-      links: %{"GitHub" => @source_url}
+      links: %{
+        "GitHub" => @source_url,
+        "Changelog" => "https://hexdocs.pm/incident_io/changelog.html"
```

Comment:
This link will resolve correctly after the next version of this package is released.